### PR TITLE
FAT2-87 keyword search

### PR DESCRIPTION
**Pull request by scottywakefield**

### CODE REVIEW

File: src/SFA.DAS.Courses.Domain/Entities/Standard.cs [hunk 1]
```diff
@@ -1,7 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
 namespace SFA.DAS.Courses.Domain.Entities
 {
     public class Standard : StandardBase
     {
+        [NotMapped]
```
Comment (dashton82) [inline src/SFA.DAS.Courses.Domain/Entities/Standard.cs:7]: Can this attribute be moved to the entity configuration instead? Just so its all in one place

File: src/SFA.DAS.Courses.Application/Courses/Queries/GetStandardsList/GetStandardsListQueryHandler.cs [hunk 2]
```diff
@@ -16,11 +17,15 @@ public GetStandardsListQueryHandler(IStandardsService standardsService)
 
         public async Task<GetStandardsListResult> Handle(GetStandardsListQuery request, CancellationToken cancellationToken)
         {
-            var standards = await _standardsService.GetStandardsList();
+            var standards = (await _standardsService.GetStandardsList(request.Keyword))
```
Comment (dashton82) [inline src/SFA.DAS.Courses.Application/Courses/Queries/GetStandardsList/GetStandardsListQueryHandler.cs:20]: Can this be put into a Task.Whenall() instead?

File: src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandardsList.cs [hunk 3]
```diff
@@ -19,25 +19,29 @@ public class WhenCallingGetStandardsList
...
         {
             mockMediator
                 .Setup(mediator => mediator.Send(
-                    It.IsAny<GetStandardsListQuery>(), 
+                    It.Is<GetStandardsListQuery>(query => query.Keyword == keyword), 
                     It.IsAny<CancellationToken>()))
                 .ReturnsAsync(queryResult);
 
-            var controllerResult = await controller.GetList() as ObjectResult;
+            var controllerResult = await controller.GetList(keyword) as ObjectResult;
```
[the remark this answers is not in the document]
Comment (scottywakefield) [reply src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandardsList.cs:33]: oh yeah for sure.

File: src/SFA.DAS.Courses.Application/Courses/Queries/GetStandardsList/GetStandardsListQueryHandler.cs (hunk 2 above)
Comment (scottywakefield) [reply src/SFA.DAS.Courses.Application/Courses/Queries/GetStandardsList/GetStandardsListQueryHandler.cs:20]: yes

**Pull request merged**